System/Console/ANSI/Example.hs

```diff
@@ -264,4 +266,121 @@ titleExample = do
     pause
     -- Yup, I'm a new title! - ansi-terminal-ex - 83x70
     ---------------------------------------------------
-    -- Title Demo
\ No newline at end of file
+    -- Title Demo
+
+codesExample :: IO ()
+codesExample = do
```

Comment:
Right -- this makes sense as a test program, but not as an example (which is intended for users, not testers or developers).